### PR TITLE
OBSDOCS-1229: Correct the name of the mentioned exporter

### PR DESCRIPTION
**Pull request by max-cx**
Target branch: `main`

:warning: Peer reviewer, please ignore the first peer review, which was requested by mistake (https://github.com/openshift/openshift-docs/pull/80452#issuecomment-2338392844).

Version(s):

4.12, 4.13, 4.14, 4.15, 4.16, 4.17

Issue:

https://issues.redhat.com/browse/OBSDOCS-1229

Link to docs preview:

https://80452--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-collector/otel-collector-exporters.html#debug-exporter_otel-collector-exporters

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: observability/otel/otel-collector/otel-collector-exporters.adoc [hunk 1]
```diff
@@ -91,15 +91,21 @@ The Debug Exporter prints traces and metrics to the standard output.
...
         traces:
-          exporters: [logging]
+          exporters: [debug]
         metrics:
-          exporters: [logging]
+          exporters: [debug]
 # ...
 ----
 <1> Verbosity of the debug export: `detailed` or `normal` or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
+<2> "number of messages initially logged each second" The default value is `2` seconds.
```
Comment (kowen-rh) [inline observability/otel/otel-collector/otel-collector-exporters.adoc:106]:
```adoc
<2> The number of messages initially logged each second. The default value is `2`.
```

Is the default value two seconds or two messages?

File: observability/otel/otel-collector/otel-collector-exporters.adoc [hunk 2]
```diff
@@ -91,15 +91,21 @@ The Debug Exporter prints traces and metrics to the standard output.
...
+<3> "sampling rate after the initial messages are logged (every Mth message is logged)"
```
Comment (kowen-rh) [inline observability/otel/otel-collector/otel-collector-exporters.adoc:107]:
```adoc
<3> The sampling rate after logging the initial messages. Logs each Nth message.
```

What's the default value?

File: observability/otel/otel-collector/otel-collector-exporters.adoc [hunk 3]
```diff
@@ -91,15 +91,21 @@ The Debug Exporter prints traces and metrics to the standard output.
...
+<4> "uses the collector's internal logger for output" The default value is `true`. "the exporter uses the collector's internal logger for output"
```
Comment (kowen-rh) [inline observability/otel/otel-collector/otel-collector-exporters.adoc:108]:
```adoc
<4> If `true`, the exporter uses the internal logger of the collector for output. The default value is `true`.
```

**Review by kowen-rh — CHANGES_REQUESTED** [review]
/remove-label peer-review-in-progress
/label peer-review-done

File: observability/otel/otel-collector/otel-collector-exporters.adoc (hunk 1 above)
Comment (IshwarKanse) [reply observability/otel/otel-collector/otel-collector-exporters.adoc:106]: The parameter sets the number of messages logged not the value for the time period. 
sampling_initial (default = 2): number of messages initially logged each second.

File: observability/otel/otel-collector/otel-collector-exporters.adoc (hunk 2 above)
Comment (IshwarKanse) [reply observability/otel/otel-collector/otel-collector-exporters.adoc:107]: sampling_thereafter (default = 1): sampling rate after the initial messages are logged (every Mth message is logged). The default value of 1 means that sampling is disabled. To enable sampling, change sampling_thereafter to a value higher than 1. Refer to [Zap docs](https://godoc.org/go.uber.org/zap/zapcore#NewSampler) for more details on how sampling parameters impact number of messages.

**Comment by max-cx** [comment]
@kowen-rh, hi, Kevin, :sweat_smile: I don't remember how it came about that I requested a peer review on this PR, but looking at it now this was clearly not ready for a peer review. Either I asked for a peer review in the wrong browser tab or there was unexpected commit amend. Again, my apologies.

File: observability/otel/otel-collector/otel-collector-exporters.adoc (hunk 2 above)
Comment (max-cx) [reply observability/otel/otel-collector/otel-collector-exporters.adoc]: Added. Thank you both.

File: observability/otel/otel-collector/otel-collector-exporters.adoc (hunk 1 above)
Comment (max-cx) [reply observability/otel/otel-collector/otel-collector-exporters.adoc]: Thank you both.

**Review by IshwarKanse — APPROVED** [review]

File: observability/otel/otel-collector/otel-collector-exporters.adoc [hunk 4]
```diff
@@ -91,15 +91,21 @@ The Debug Exporter prints traces and metrics to the standard output.
...
+          exporters: [debug]
         metrics:
-          exporters: [logging]
+          exporters: [debug]
 # ...
 ----
-<1> Verbosity of the debug export: `detailed` or `normal` or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
+<1> Verbosity of the debug export: `detailed`, `normal`, or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
+<2> Initial number of messages logged per second. The default value is `2` messages per second.
+<3> Sampling rate after the initial number of messages has been logged. Disabled by default with the default `1` value. Sampling is enabled with values greater than `1`. For more information, see the page for the link:https://pkg.go.dev/go.uber.org/zap/zapcore?utm_source=godoc#NewSampler[`zapcore`] package on the Go Project's website.
```
Comment (mburke5678) [inline observability/otel/otel-collector/otel-collector-exporters.adoc:107]: > the initial number of messages
How do I know what this value is?

Comment (mburke5678) [reply observability/otel/otel-collector/otel-collector-exporters.adoc:107]: Maybe specify the function that you are using so the use knows s/he is in the right place on the page?
```suggestion
<3> Sampling rate after the initial number of messages has been logged. Disabled by default with the default `1` value. Sampling is enabled with values greater than `1`. For more information, see the link:https://pkg.go.dev/go.uber.org/zap/zapcore?utm_source=godoc#NewSampler[`func NewSampler` ] in the zapcore package on the Go Project's website.
```

Comment (max-cx) [reply observability/otel/otel-collector/otel-collector-exporters.adoc:107]: > How do I know what this value is?
Good question. It's actually the value in the previous callout. I've added it as an appositive to make sure every reader understands. Thank you for raising this point.

**Comment by mburke5678** [comment]
@max-cx LGTM. For a future project, the codeblocks throughout the assembly [need the `apiVersion` and `kind`, per the doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields).

**Pull request merged**